OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -110,6 +114,18 @@ public Carryall(Actor self, CarryallInfo info)
 			move = self.Trait<IMove>();
 			facing = self.Trait<IFacing>();
 			this.self = self;
+
+			if (!string.IsNullOrEmpty(info.InitialActor))
```

Comment:
Will `FieldLoader` strip the value of leading & trailing whitespace (before we get to this point in the code of coursee), so we don't need to check for whitespace-only here?

Reply:
It does.